src/Web/Slack/Reaction.hs

```diff
@@ -0,0 +1,14 @@
+module Web.Slack.Reaction (addReaction) where
+
+import           Control.Lens
+import           Control.Monad.Except
+import qualified Data.Text           as T
+import           Web.Slack.State
+import           Web.Slack.Types
+import           Web.Slack.WebAPI
+
+-- | Add a reaction to a message from the specified channel.
+addReaction :: ChannelId -> T.Text -> SlackTimeStamp -> Slack s ()
+addReaction cid emoji timestamp = do
+  cfg <- use config
+  void . liftIO . runExceptT $ reactions_add_message cfg cid emoji timestamp
```

Comment:
Using `void` here will swallow up any errors, which may not be desirable for all users.  In my use case, I wanted to add reaction emojis regardless of whether or not I had previously added a reaction, and there is a specific error associated with trying to add a reaction when you already added it before.

This was a quick approach rather than refactoring `makeSlackCall`.

Reply:
Let's definitely not throw away errors.